invenio_communities/views/communities.py

```diff
@@ -57,6 +61,27 @@
 ]
 
 
+def render_branded_template(template_name_or_list, theme_brand=None, **context):
```

Comment:
You might want to harmonize naming to use `theme` community, instead of branding.